assets/README.powershellcommunity.md

```diff
@@ -0,0 +1,91 @@
+  * Example: `docker pull pshorg/powershellcommunity:oraclelinux-7.5`
+* **[Parrot Linux][parrotsec-linux-uri]**
+  * Tags: `parrot-4.2.2` , `6.1.1-parrot-4.2.2`
+  * Dockerfile: [/release/community-stable/parrot/docker/Dockerfile][parrotsec-linux-stable-dockerfile]
+  * Example: `docker pull pshorg/powershellcommunity:parrot-4.2.2`
+
+[amazon-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/amazonlinux/docker/Dockerfile
+[arch-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/archlinux/docker/Dockerfile
+[blackarch-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/blackarch/docker/Dockerfile
+[clear-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/clearlinux/docker/Dockerfile
+[kali-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/kali-rolling/docker/Dockerfile
+[oracle-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/oraclelinux/docker/Dockerfile
+[parrotsec-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/parrot/docker/Dockerfile
+
+[amazon-linux-uri]: https://aws.amazon.com/amazon-linux-2/
+[arch-linux-uri]: https://www.archlinux.org/
+[blackarch-linux-uri]: https://www.blackarch.org/
+[clear-linux-uri]: https://www.clearlinux.org/
+[kali-linux-uri]: https://www.kali.org/
+[oracle-linux-uri]: https://www.oracle.com/linux/
+[parrotsec-linux-uri]: https://www.parrotsec.org/
+
+## Legal and licensing
+
+PowerShell is licensed under the [MIT license][].
+
+[MIT license]: https://github.com/PowerShell/PowerShell/tree/master/LICENSE.txt
+
+[Third-Party Software Notices and Information](https://github.com/PowerShell/PowerShell/blob/master/ThirdPartyNotices.txt)
+
+## About the image
+
+PowerShell Core is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework. It works well with your existing tools and is optimized for dealing with structured data (for example, JSON, CSV, and XML), REST APIs, and object models.
+It includes a command-line shell, an associated scripting language, and a framework for processing cmdlets.
+
+If you are new to PowerShell and want to learn more, see the [getting started][] documentation.
+
+[getting started]: https://github.com/PowerShell/PowerShell/tree/master/docs/learning-powershell
+
+## How to use this image
+
+See our [Docker examples](https://github.com/PowerShell/PowerShell/tree/master/docker#examples).
+
+## Issues
+
+If you have any problems with or questions about this image, please contact us through a [GitHub issue][].
+
+[GitHub issue]: https://github.com/PowerShell/PowerShell-Docker/issues
+
+## Related
+
+PowerShell Docker Hub repo:
```

Comment:
The above link is correctly the docker issues.  This one is the link to product issues.  I didn't check if this is broken in the other readme.